WzComparerR2/CharaSimControl/GearTooltipRender2.cs

```diff
@@ -399,7 +399,15 @@ private Bitmap RenderBase(out int picH)
                 bool max = (Gear.Levels != null && value >= Gear.Levels.Count);
                 TextRenderer.DrawText(g, "성장 레벨 : " + (max ? "MAX" : value.ToString()), GearGraphics.EquipDetailFont, new Point(13, picH), ((SolidBrush)GearGraphics.OrangeBrush3).Color, TextFormatFlags.NoPadding);
                 picH += 15;
-                TextRenderer.DrawText(g, "성장 경험치" + (max ? ": MAX" : " : 0%"), GearGraphics.EquipDetailFont, new Point(13, picH), ((SolidBrush)GearGraphics.OrangeBrush3).Color, TextFormatFlags.NoPadding);
+
+                if (Gear.Levels != null)
+                {
+                    TextRenderer.DrawText(g, "성장 경험치" + (max ? ": MAX" : " : 0/" + Gear.Levels.First().Point), GearGraphics.EquipDetailFont, new Point(13, picH), ((SolidBrush)GearGraphics.OrangeBrush3).Color, TextFormatFlags.NoPadding);
+                }
+                else
+                {
+                    TextRenderer.DrawText(g, "성장 경험치" + (max ? ": MAX" : " : 0%"), GearGraphics.EquipDetailFont, new Point(13, picH), ((SolidBrush)GearGraphics.OrangeBrush3).Color, TextFormatFlags.NoPadding);
+                }
```

Comment:
```suggestion
                string expString = Gear.Levels != null ? " : 0/" + Gear.Levels.First().Point : " : 0%";
                TextRenderer.DrawText(g, "성장 경험치" + (max ? ": MAX" : expString), GearGraphics.EquipDetailFont, new Point(13, picH), ((SolidBrush)GearGraphics.OrangeBrush3).Color, TextFormatFlags.NoPadding);
```

조금 복잡하긴 하지만 짧게 가봅시다.